src/engine/x86-64/X86_64MacroAssembler.v3

```diff
@@ -21,11 +21,19 @@ class X86_64MacroAssembler extends MacroAssembler {
 	var jump_tables: Vector<(int, Array<X86_64Label>)>;
 	var offsets: V3Offsets;
 	var trap_stubs: X86_64SpcTrapsStub;
+	var last_code_print_at = 0;
 
 	new(w, regConfig: RegConfig) super(Target.tagging, regConfig) {
 		scratch = G(regConfig.scratch);
 	}
-
+	def printCodeBytes(sb: StringBuilder) {
```

Comment:
This adds a little more local state; how about keeping track of the offset of the bytecode start in the SPC? Partially because we should consider saving that for use at runtime, e.g. to compute the PC at frame walking instead of storing it as is done now.